galaxy.yml

```diff
@@ -0,0 +1,102 @@
+  - Sandeepyadav93 (https://github.com/Sandeepyadav93)
+  - jpodivin (https://github.com/jpodivin)
+  - lewisdenny (https://github.com/lewisdenny)
+  - afazekas (https://github.com/afazekas)
+  - arxcruz (https://github.com/arxcruz)
+  - fmount (https://github.com/fmount)
+  - bshewale (https://github.com/bshewale)
+  - dasm (https://github.com/dasm)
+  - frenzyfriday (https://github.com/frenzyfriday)
+  - cescgina (https://github.com/cescgina)
+  - bshephar (https://github.com/bshephar)
+  - steveb (https://github.com/steveb)
+  - slagle (https://github.com/slagle)
+  - rebtoor (https://github.com/rebtoor)
+  - dsariel (https://github.com/dsariel)
+  - son-vyas (https://github.com/son-vyas)
+  - tosky (https://github.com/tosky)
+
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
+description: "CI-Framework is a way to bootstrap development and CI environments for RHOSO."
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license: []
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: 'LICENSE'
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags: []
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies:
+  'git+https://github.com/containers/ansible-podman-collections': 'master'
+  'git+https://github.com/ansible-collections/community.general': 'main'
+  'git+https://github.com/ansible-collections/ansible.posix': 'main'
+  'git+https://github.com/ansible-collections/community.libvirt': 'main'
+  'git+https://github.com/ansible-collections/community.crypto': 'main'
+  'git+https://github.com/ansible-collections/kubernetes.core': 'main'
+  'git+https://github.com/openstack/ansible-collections-openstack': 'master'
+  'git+https://github.com/ansible-collections/ansible.netcommon': 'main'
+  'git+https://github.com/openstack/ansible-config_template': 'master'
+
+# The URL of the originating SCM repository
+repository: https://github.com/openstack-k8s-operators/ci-framework
+
+# The URL to any online docs
+documentation: https://ci-framework.readthedocs.io/en/latest/
+
+# The URL to the homepage of the collection/project
+homepage: https://github.com/openstack-k8s-operators/ci-framework
+
+# The URL to the collection issue tracker
+issues: https://github.com/openstack-k8s-operators/ci-framework/issues
```

Comment:
maybe better to point to jira (or to the doc page since we explain how to report an issue)